packages/medusa/src/utils/build-query.ts

```diff
@@ -153,7 +153,7 @@ function buildWhere<TWhereKeys extends object, TEntity>(
  * output: ['test.test1', 'test.test2', 'test.test3.test4', 'test2']
  * @param input
  */
-export function buildLegacyFieldsListFrom<TEntity>(
+export function objectToStringPath<TEntity>(
```

Comment:
suggestion: Keep this and deprecate it to stay backward compatible. And change all occurrences to import from `@medusajs/utils`